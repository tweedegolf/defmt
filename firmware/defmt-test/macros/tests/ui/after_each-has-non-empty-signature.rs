fn main() {}

#[defmt_test_macros::tests]
mod tests {
    #[init]
    fn init() {
        // empty
    }

    #[after_each]
    fn test(arg: &mut u8) {
        assert!(true);
    }
}
