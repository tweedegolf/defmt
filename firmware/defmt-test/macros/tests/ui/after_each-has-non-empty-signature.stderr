error: no state was initialized by `#[init]`; signature must be `fn()`
  --> tests/ui/after_each-has-non-empty-signature.rs:11:8
   |
11 |     fn test(arg: &mut u8) {
   |        ^^^^
